apps/dashboard/src/app/resources/resource-list.component.ts

```diff
@@ -105,6 +108,23 @@ export class ResourceListComponent implements OnInit, OnDestroy {
   labelSets$ = this.sdk.currentKb.pipe(switchMap((kb) => kb.getLabels()));
   currentLabelList: Classification[] = [];
 
+  viewerWidget = this.sdk.currentKb.pipe(
+    distinctUntilKeyChanged('id'),
+    tap(() => {
+      document.getElementById('viewer-widget')?.remove();
+    }),
+    map((kb) =>
+      this.sanitized.bypassSecurityTrustHtml(`<nuclia-viewer id="viewer-widget"
```

Comment:
I have problems when the kb is private, I get 403

Reply:
ok, let me have a look